Find a functionally equivalent code:android.net.NetworkTemplate.matchesMobile:COMMENT
Method Modifier: private     
Comment:/**
 * Check if mobile network with matching IMSI.
 */

@@ -3,8 +3,8 @@
         // TODO: consider matching against WiMAX subscriber identity
         return true;
     } else {
-        return ((sForceAllNetworkTypes || contains(DATA_USAGE_NETWORK_TYPES, ident.mType)) && Objects.equal(mSubscriberId, ident.mSubscriberId));
+        return ((sForceAllNetworkTypes || contains(DATA_USAGE_NETWORK_TYPES, ident.mType)) && Objects.equals(mSubscriberId, ident.mSubscriberId));
     }
 }

